scrapinghub/utils.py

```diff
@@ -4,22 +4,38 @@
 from six import string_types
 from requests.exceptions import HTTPError
 
-from scrapinghub import APIError
 
+class ScrapinghubAPIError(Exception):
 
-class DuplicateJobError(APIError):
+    def __init__(self, origin):
```

Comment:
should be possible to raise `ScrapinghubAPIError` without origin and with some custom error message